scripts/west_commands/runners/jlink.py

```diff
@@ -99,7 +99,7 @@ def do_add_parser(cls, parser):
                             action=partial(depr_action,
                                            replacement='-i/--dev-id'),
                             help='Deprecated: use -i/--dev-id instead')
-        parser.add_argument('--remote', required=False, dest='ip',
+        parser.add_argument('--ip', required=False, dest='ip',
```

Comment:
This is a fixup commit, which needs to be squashed properly. Please watch the youtube video I linked; it explains this as well.